src/tests/Sort.test.js

```diff
@@ -0,0 +1,46 @@
+const { app } = require('../pages/Application'); 
+
+
+describe('Inventory Sorting', () => {
+    before(async () => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
+    });
+
+    it('should sort items by name (Z to A)', async () => {
+        await app.inventory.navigate();
+
+        const inventoryItems = await app.inventory.inventoryItems;
+        if (!inventoryItems.length) {
+            throw new Error('No inventory items found on the page');
+        }
+
+        const initialNames = [];
+        for (const item of inventoryItems) {
+            const itemName = await item.$('.inventory_item_name').getText();
+            initialNames.push(itemName);
+        }
+
+        console.log('Initial names:', initialNames);
+
+        const sortDropdown = await $('select.product_sort_container');
+        if (!await sortDropdown.isDisplayed()) {
+            throw new Error('No inventory items found on the page');
+        }
+        await sortDropdown.selectByAttribute('value', 'za');
+
+        const sortedNames = [];
+        for (const item of inventoryItems) {
+            const itemName = await item.$('.inventory_item_name').getText();
+            sortedNames.push(itemName);
+        }
+
+        console.log('Sorted names:', sortedNames);
+
+        const expectedSortedNames = [...initialNames].sort();
+
+        console.log('Expected Sorted Names:', expectedSortedNames);
+    }
```

Comment:
U need to create other tests for sorting options like z-a hi-lo lo-hi. 

So, try to figure out the way to  isolate the logic and create a method which u can give an array and sorting option to get an expected result (inside Inventory.page.js)

Reply:
I have created a method in the Inventory.js file, not myself, peeped a lot, I hope I understood the remark correctly. 🤞